### DISCUSSION TITLE
Should we install a stale-bot to this repo?

### DISCUSSION THREAD

**Discussion by nknapp** [handlebars-lang/handlebars.js#1867] [Ideas]
@papasmile suggested installing https://github.com/marketplace/stale on this repo in #1652.
We can discuss this here.

**Comment by nknapp** [comment]
A stale-bot may reduce the number of open issues. But the problem with Handlebars is that a lot of issues simply remain unanswered. I don't have the resources to answer every single one and there are almost no other maintainer who have resources...

It seems wrong to automatically close issues, just because noone answers the initial question. If the issue-author does not respond to questions or proposals, that is another matter.

I think, what we need in this repo is a bit of triaging.

**Comment by papasmile** [comment]
Stale-bot will easily allow you to ignore certain labels.  You could mark any issues you think need attention with a special label (e.g. 'needs answer') then turn it on...

**Comment by ErisDS** [comment]
I am very strongly in favour of adding a stale bot. Having lots of open issues is noise that actively gets in the way of contributions. It's impossible to tell what's important and what's not. 

I hear what you're saying about questions, but we can solve that easily IMO by adding issue templates. I would add one for questions that strongly encourages people to post usage questions on stackoverflow for the simple reason that they tend to get answered there cos that's where the community of people who want to answer questions go.

GitHub is more oriented towards maintainers, contributors and people with bugs or feature requests. IMO everyone has a better experience when these two things (maintenance and support) are separate.

That being said, I think we can and should use a very slow/lenient bot with nice big timeframes cos this is a fairly slow moving ship and that's not a bad thing 🙂

**Comment by ErisDS** [comment]
FYI another side effect of adding a stale bot would be getting community members to chime in again on any of their issues that are breaking changes that we should take time to handle before releasing master.
